File: Foundation/Model/About/About.cs
using ServiceStack.DataAnnotations;


namespace Foundation.Model.About
{

    public class About
    {
        [AutoIncrement]
        public int Id { get; set; }

        public string Title { get; set; }

        public string Content1 { get; set; }

        public string Content2 { get; set; }

        public string Content3 { get; set; }

        public string Lang { get; set; }

        public string ImageUrl1 { get; set; }

        public string ImageUrl2 { get; set; }

        public string ImageUrl3 { get; set; }
    }
}

File: AdminKP/Views/About/About.js
var About = {
    init: function () {
        $('#dvAbout').jtable({
            title: 'About',
            paging: true,
            sorting: false,
            actions: {
                listAction: '/About/GetAbout',
                deleteAction: '/About/GetAbout',
                updateAction: '/About/GetAbout',
                createAction: '/About/GetAbout'
            },

            fields: {
                Id: {
                    key: true,
                    create: false,
                    edit: false,
                    list: false
                },

                Title: {
                    title: '',
                },

                Content1: {
                    title: '',
                },

                Content2: {
                    title: '',
                    width: '17%',
                },
                
                Content3: {
                    title: '',
                    width: '17%',
                },
                
                Lang: {
                    title: '',
                    width: '17%',
                }
            }
        });

        //Load student list from server
        $('#dvEtsoCodeGrid').jtable('load');
    }
};

$(document).ready(function () {
    EtsoCode.init();
});